newton/tests/test_model.py

```
@@ -325,11 +325,17 @@ def __init__(self, vertices, faces):
         self.assertAlmostEqual(model.approx_attr.numpy()[extra_shape], shape_attr, places=6)
 
     def test_approximate_meshes_collision_filter_child_bodies(self):
-        def normalize_pair(a, b):
-            return (min(a, b), max(a, b))
+        def normalize_pair(builder, a, b):
+            t1 = builder.shape_type[a]
+            t2 = builder.shape_type[b]
+            if t1 > t2:
+                return (b, a)
+            elif t1 == t2 and s1 > s2:
```

Ruff (0.15.11)

[error] Undefined name `s1`

(F821)